readthedocs/projects/models.py

```diff
@@ -202,6 +202,14 @@ class Project(models.Model):
         ),
     )
 
+    external_builds_enabled = models.BooleanField(
+        _('Build pull requests for this project'),
+        default=False,
+        # TODO: Remove this after migrations
+        null=True,
```

Comment:
This is not needed when using a `default=` value. I'd remove it in this PR instead of forcing us to create a new one later.

Reply:
That only works on the new code though, when we run the migration and the old code is still running, it will blow up the DB.